src/urban/events/migrations.py

```
@@ -75,3 +75,22 @@ def install_housing_procedure(context):
             base_json_path="./profiles/config/standard/housing",
             handle_existing_content=utils.ExistingContent.UPDATE,
         )
+
+
+def install_bordering_licence_events(context):
```

Ruff (0.15.5)

[warning] Unused function argument: `context`

(ARG001)